docs/zh/connector-v2/source/HdfsFile.md

```diff
@@ -10,15 +10,15 @@
 
 ## 主要特性
 
-- [x] [批处理](../../../en/concept/connector-v2-features.md)
-- [ ] [流处理](../../../en/concept/connector-v2-features.md)
-- [x] [精确一次](../../../en/concept/connector-v2-features.md)
+- [x] [批处理](../../concept/connector-v2-features.md)
+- [ ] [流处理](../../concept/connector-v2-features.md)
+- [x] [精确一次](../../concept/connector-v2-features.md)
 
 在一次 pollNext 调用中读取分片中的所有数据。将读取的分片保存在快照中。
 
-- [x] [列投影](../../../en/concept/connector-v2-features.md)
-- [x] [并行度](../../../en/concept/connector-v2-features.md)
-- [ ] [支持用户定义的分片](../../../en/concept/connector-v2-features.md)
```

Comment:
Please do not change this. It work fine in website. You can refer https://seatunnel.apache.org/docs/2.3.4/connector-v2/source/HdfsFile

Reply:
Thank you for your review. The website is an English document, will my modifications to the Chinese document also have an impact?

Reply:
Oh, sorry about that. I missed this is chinese file. Are you verified in local with this change? Does it work fine?

Reply:
Yes, I replaced the associated document link here with the translated Chinese document link.